tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3874,4 +3874,42 @@ function data_set_spacing_sizes_when_invalid() {
 			),
 		);
 	}
+
+	/**
```

Comment:
I very much welcome this data provider and the additional test.

As a convention, data providers are normally directly _after_ the test though, not _before_.

Reply:
My bad. Thanks for pointing it out.
Fixed in 0d66f734d39e7013a5d31a5916fd1847f88bcc04

Reply:
No worries, you can't know everything. Thanks for changing the method order!